lib/pages/item_request.dart

```diff
@@ -113,34 +111,18 @@ class _ItemRequestState extends State<ItemRequest> {
     );
   }
 
-  Future<void> addLost() {
-    // Calling the collection to add a new user
-    return lost
-        //adding to firebase collection
-        .add({
-      //Data added in the form of a dictionary into the document.
-      'Date Created': Timestamp.now(),
-      'Description': description,
-      'ItemName': category,
-      'Location': GeoPoint(lat, long),
-      'Phone': phone,
-      'Picture': _imgName
-    });
+  Future<void> addLost() async {
+    Item item = Item(id: "", userId: AuthService.instance.user!.uid, timeCreated: Timestamp.now(), description: description, itemName: category, location: GeoPoint(lat, long),phone: phone, picture: _imgName);
+    DocumentReference ref = await FirebaseFirestore.instance.collection('lost').add(item.toFirestore());
```

Comment:
Current implementation has collection name being 'Lost' and not 'lost', we need to come to consensus how we name these.

Reply:
I standardized everything as a part of this PR - forgot to mention that. Everything is camelCase